modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -1609,6 +1609,7 @@ api-endpoint=API Endpoint
 api-endpoint-description=Set the endpoint for the Computer Vision API V2 (e.g. {0}). See {1} for more information.
 api-key=API Key
 api-login-id=API Login ID
+api-schema-changes-were-saved=Api schema changes were saved.
```

Comment:
@cgmonte Shouldn't it be `API schema...`?

Reply:
Yes, it should! Thanks :)